spec/src/main/asciidoc/configexamples.asciidoc

```diff
@@ -466,18 +471,6 @@ clientDetails.port -> client.port -> 9081
 clientDetails.endpoint -> client.endpoint -> shelf
 clientDetails.getLocation() -> client.old.location -> Dublin
 ```
-If `@ConfigProperties` has no associated prefix, it means the prefix is empty. e.g.
```

Comment:
can you add the condition when prefix is empty, e.g. `@ConfigProperties(prefix="")`